src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentaion/InputFastMatcher.java

```diff
@@ -1,4 +1,4 @@
-package com.google.checkstyle.test.chapter4formatting.rule42blockindentaion;
+package com.google.checkstyle.test.chapter4formatting.rule42blockindentation;
 
```

Comment:
The folder should be renamed too.

Reply:
Done, sorry, my tool for renames handles files but didn't consider directories. I'll fix that...

Reply:
fwiw, although GitHub doesn't seem to recognize that this comment chain is obsolete, it is.